openedx/adg/admin_dashboard/webinars/__init__.py

```diff
@@ -1,3 +1,17 @@
 """
-App for managing webinars in admin dashboard
+This app adds the ability to manage webinars in admin dashboard.
+
+ - An admin can create a webinar and then save it.
+ - Admin needs to fill information required for a webinar and can invite participants like cohosts, panelists, invitees
+ and can also invite all the users of platform or a business unit. A business unit can invite all users or users
+ from their own business unit.
+ - Admin can then publish a webinar, publishing the webinar sends invitations to all the participants and
+ creates a webinar on zoom via zoom API. Once a webinar is published users can see it in their LEARN page and
+ can register in a webinar.
+ - Admin can update a webinar. On webinar update all the registered users and team members receive webinar update
+ emails, unregistered users receive webinar invites.
+ - An admin can cancel a webinar, Cancelling a webinar sends cancellation emails to all the registered users and
+team members. Cancelling a webinar also deletes the webinar from zoom via API.
```

Comment:
team members ~. Cancelling a webinar also~ and deletes the webinar from zoom via API.